apps/language_server/test/server_test.exs

```diff
@@ -937,6 +937,8 @@ defmodule ElixirLS.LanguageServer.ServerTest do
 
       # File is already formatted
       assert response(3, []) == resp
+
+      wait_until_compiled(server)
```

Comment:
Instead of adding these to each individual test directly, could we add it to a generic setup block? Perhaps one that checks for `fixture: true`. That would make it harder to miss one of these in the future.

Reply:
I tried adding it in `on_exit` callback but the server process is already down when it's executed. We'd need wrapping it in a lambda or macro with `do` block.

Reply:
Ah, k.